src/app/common-hybrid-angular.module.ts

```diff
@@ -104,8 +107,11 @@ export class EmptyComponent {}
     NodeIconComponent,
     NodeStatusIcon,
     OpenResponseGrading,
+    PeerChatChatBoxComponent,
     PeerChatMessageComponent,
     PeerChatMessagesComponent,
+    PeerChatPreviousWorkComponent,
+    PeerChatQuestionBankComponent,
```

Comment:
Move the PeerChat*Component into its own module? See GK's similar work here: https://github.com/WISE-Community/WISE-Client/pull/408/commits/b9612cb7c6a348612d3cd2c720ba54e8e314b3ce